documentation/docs/07-hooks.md

```diff
@@ -79,47 +115,37 @@ export async function handle({ event, resolve }) {
 }
 ```
 
-### handleError
+#### handleError
 
-If an error is thrown during loading or rendering, this function will be called with the `error` and the `event` that caused it. This allows you to send data to an error tracking service, or to customise the formatting before printing the error to the console.
-
-During development, if an error occurs because of a syntax error in your Svelte code, a `frame` property will be appended highlighting the location of the error.
+If an error is thrown during loading or rendering, this function will be called with the `error` and the `event` that caused it. This allows for two things:
 
-If unimplemented, SvelteKit will log the error with default formatting.
+- you can send data to an error tracking service, for example
+- you can customise the formatting. The return value of the function becomes the value of `$page.error`. To make this type-safe, you can customize the expected shape through `App.PageError`. Note that `message` is always a required property, to guarantee a sensible fallback behavior.
```

Comment:
```suggestion
- you can customise the formatting. The return value of the function becomes the value of `$page.error`. To make this type-safe, you can customize the expected shape through `App.PageError`, which must include a `message: string` property.
```

Reply:
I liked the previous version better because it also explains why.